src/utils.py

```diff
@@ -2,10 +2,12 @@
 
 import daiquiri
 import requests
+import re
 
 from src.config import DAIQUIRI_LOG_LEVEL
 from src.gremlin import execute_query
 
+CVE_REGULAR_EXPRESSION = r"CVE-\d{4}-\d{4,}"
```

Comment:
Are you sure that second part will be always 4 digits? `\d{4}`

Reply:
Its `\d{4,}` so 4 or more digit we are expecting.

Reply:
In the format CVE-X-Y, I had mentioned about the X part. So the X always denotes year?